packages/mongo/mongo_driver.js

```diff
@@ -834,25 +834,32 @@ MongoConnection.prototype.find = function (collectionName, selector, options) {
     self, new CursorDescription(collectionName, selector, options));
 };
 
-MongoConnection.prototype.findOne = function (collection_name, selector,
```

Comment:
this is correct?

Reply:
Yeah, I moved the declaration for the `fibers` one below

Reply:
oh, okay. 
Why are the tests not passing?

Reply:
@Grubba27 Fixed the tests here, but [this one](https://app.circleci.com/pipelines/github/meteor/meteor/5158/workflows/7037d89a-ecee-41cc-a9e4-e72cb1695a0e/jobs/102685) is failing sometimes. Seems to be some instability with maven - I don't think it's related to the changes on this PR, what do you think? Any ideas?

Reply:
Now it have passed. Will merge this PR.